main.cs

```diff
@@ -17,6 +18,28 @@ public class Programa{
         Azul.Add(new Aldeano());
         Azul.Add(new Guerrero());
         Azul.Add(new Arquero());
+        while(/*Aqui se comprobaria si algun equipo ha ganado, comprobando si alguna unidad tiene mas de 0 de vida*/ true){
```

Comment:
**Error:** No pongáis comentarios en el medio del código, únicamente dificulta la lectura del código a alguien que lo revisa, un despiste así puede costar muy caro, y más si estamos hablando de algo delicado como es un `while(true)` quedaría mejor algo del estilo:
`while(true) // TODO: Cambiar true por condicion para mantener el sistema de turnos de la partida`